### PR TITLE
DIO-1719: Added funcitonality to dorel-bynder-image. Added image to i…

### PR DESCRIPTION
**Pull request by petercuret**
…nfo templating.

### CODE REVIEW

File: src/atoms/dorel-bynder-image.html [hunk 1]
```diff
@@ -167,24 +208,31 @@
...
         if (typeof this.mediaContent !== 'undefined' && this.mediaContent.hasOwnProperty('thumbnails')) {
-          // Set the Bynder description to this component's alt properties
-          this.alt = this.mediaContent.description;
-
-          // Default to the root media thumbnail and webimage
-          if (this.derivativeIdentifier === '') {
-            this.imagePlaceholder = this.mediaContent.thumbnails.mini;
-            this.imageSource = this.mediaContent.thumbnails.webimage;
+          // Set the Bynder description to this component's alt properties, if no alt is given
+          if(this.alt === '')
```
Comment (aronmartin) [inline src/atoms/dorel-bynder-image.html]: use this.get - polymer convention

File: src/atoms/dorel-bynder-image.html [hunk 2]
```diff
@@ -167,24 +208,31 @@
...
+          {
+            this.alt = this.mediaContent.description;
```
Comment (aronmartin) [inline src/atoms/dorel-bynder-image.html]: use this.set()

File: src/atoms/dorel-bynder-image.html [hunk 3]
```diff
@@ -167,24 +208,31 @@
...
-          // Retrieve the derivative if one is specified
-          else {
-            var bynderDerivatives = this.mediaContent.thumbnails;
-            this.imagePlaceholder = bynderDerivatives.mini;
-            // Checks if the derivative exists
-            if (bynderDerivatives.hasOwnProperty(this.derivativeIdentifier)) {
-              this.imageSource = bynderDerivatives[this.derivativeIdentifier];
 
+          var bynderDerivatives = this.mediaContent.thumbnails;
+          this.imagePlaceholder = bynderDerivatives.mini;
```
Comment (aronmartin) [inline src/atoms/dorel-bynder-image.html]: this.set()

File: src/atoms/dorel-bynder-image.html [hunk 4]
```diff
@@ -167,24 +208,31 @@
...
           }
-          // Retrieve the derivative if one is specified
-          else {
-            var bynderDerivatives = this.mediaContent.thumbnails;
-            this.imagePlaceholder = bynderDerivatives.mini;
-            // Checks if the derivative exists
-            if (bynderDerivatives.hasOwnProperty(this.derivativeIdentifier)) {
-              this.imageSource = bynderDerivatives[this.derivativeIdentifier];
 
+          var bynderDerivatives = this.mediaContent.thumbnails;
```
Comment (aronmartin) [inline src/atoms/dorel-bynder-image.html]: this.get()

File: src/atoms/dorel-bynder-image.html (hunk 2 above)
Comment (petercuret) [reply src/atoms/dorel-bynder-image.html]: This is only needed if you want to observable changes, which is not the case.

File: src/atoms/dorel-bynder-image.html (hunk 1 above)
Comment (petercuret) [reply src/atoms/dorel-bynder-image.html]: Possible to use that, but it's mostly used for paths. Not needed for all property access.

**Pull request merged**